source/installation/resource-manager/linuxhost.rst

```diff
@@ -138,56 +118,108 @@ strict_host_checking
+  In order to communicate with the execution hosts the adapter uses SSH in
+  ``BatchMode``. The adapter does not take a position on whether authentication
+  is performed by user owned passwordless keys, or host-based authentication;
+  however OSC has chosen to provide `host based authentication
+  <https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Host-based_Authentication>`_
+  to its users.
+
 
-An Example User Story
-*********************
 
-As an HPC user working in ``R`` I want to be able to be able to launch RStudio so that I can use it as an IDE; if I am told that my resources are limited I will not run anything that takes up more than N CPUs, X memory, or Z hours.
+Enforce resource limits on the target host
+------------------------------------------
 
-A Non-traditional Job Launcher
-******************************
+By default the adapter does not limit the user's CPU or memory utilization, only their "walltime". The following are two examples of ways to implement resource limits for the LinuxHost Adapter using cgroups.
+
+Approach #1: Systemd user slices
+................................
 
-The LinuxHost adapter facilitates launching jobs immediately without using a traditional scheduler or resource manager. Use cases for this non-traditional job adapter include:
+With systemd it is possible to manage the resource limits of user logins through each user's `slice <https://www.freedesktop.org/software/systemd/man/systemd.slice.html>`_. The limits applied to a user slice are shared by all processes belonging to that user, this is not a per-job or per-node resource limit but a per-user limit. When setting the limits keep in mind the sum of all user limits is the max potential resource consumption on a single host.
 
-- Launching desktop environments
-- Launching code editors
-- Using OnDemand on systems that do not have a supported scheduler installed
+First update the PAM stack to include the following line:
 
-The adapter pieces together several common Linux/HPC technologies to achieve behavior similar to what a scheduler offers.
+.. code-block:: none
 
-- ``ssh`` connects from the web node to a configured host such as a login node.
-- Specially named ``tmux`` sessions offer the ability to rediscover running jobs
-- ``singularity`` containerization provides a PID namespace without requiring elevated privileges that ensures that all child processes are cleaned up when the job either times out or is killed
-- ``timeout`` is used to set a 'walltime' after which the job is killed
-- ``pstree`` is used to detect the job's parent ``sinit`` process so that it can be killed
+   session     required      pam_exec.so type=open_session /etc/security/limits.sh
```

Comment:
Should be mentioned this needs to go into a file used by the `sshd` PAM configs which on CentOS/RHEL default to `/etc/pam.d/password-auth-ac`.  Also needs to be included in the block showing position in chain as PAM order is very specific:

```
session     optional      pam_keyinit.so revoke
session     required      pam_limits.so
session     required      pam_systemd.so
session     required      pam_exec.so type=open_session /etc/security/limits.sh
session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
session     required      pam_unix.so
session     optional      pam_sss.so
```

Needs to come after `pam_systemd.so` that is also set to `required`.